exporter.go

```diff
@@ -1650,3 +1656,113 @@ func (e *Exporter) scrapeRedisHost(ch chan<- prometheus.Metric) error {
 
 	return nil
 }
+
+func (e *Exporter) extractSentinelMetrics(ch chan<- prometheus.Metric, c redis.Conn) {
+	masterDetails, err := redis.Values(doRedisCmd(c, "SENTINEL", "MASTERS"))
+	if err != nil {
+		log.Debugf("Error getting sentinel master details %s:", err)
+		return
+	}
+
+	log.Debugf("sentinel master details : %#v", masterDetails)
```

Comment:
```suggestion
	log.Debugf("Sentinel master details: %#v", masterDetails)
```

Reply:
This was updated along with other review suggestion. Thanks. :)